sdk/cosmos/azure-cosmos/CHANGELOG.md

```diff
@@ -9,6 +9,9 @@
 
 #### Bugs Fixed
 * Fixed staleness issue of `COSMOS.MIN_CONNECTION_POOL_SIZE_PER_ENDPOINT` system property - See [PR 36599](https://github.com/Azure/azure-sdk-for-java/pull/36599).
+* Fixed to not stall building of a client for the entire aggressive warm up duration but instead until containers
+in the warmup flow have been warmed up or until aggressive warm up duration has been reached whichever is lesser. - See [PR 36889](https://github.com/Azure/azure-sdk-for-java/pull/36889)
```

Comment:
Fixed an issue where cosmos client was getting blocked for the entire aggressive warmup duration even when all the connections have been opened already - See [PR 36889](https://github.com/Azure/azure-sdk-for-java/pull/36889)

Reply:
better to keep the changelog wordings simple :)

Reply:
Fixed in next iteration.